octoprint_mrbeam/static/css/tab_messages.css

```diff
@@ -1,113 +1,140 @@
+  color: #e25303;
+}
+#messages_nav_tab.has-unread-messages:hover {
+  color: #a63e03;
+}
+.mrb-messages-container {
+  padding: 25px 20px;
+}
+h1.mrb-messages-title {
+  margin: 0;
+  font-weight: bold !important;
+}
+.mrb-message-element {
+  cursor: pointer;
+  padding: 10px 15px;
+  margin-bottom: 10px;
+  border-radius: 4px;
+  border: 1px solid #e25303;
+  box-shadow: 2px 2px 4px 1px #e25303;
 }
 
-.mrb-message-element:hover{
-    background-color: transparent;
-    transform: scale(1.018);
-    transition: transform 0.3s ease;
+.mrb-message-element:hover {
+  background-color: transparent;
+  transform: scale(1.018);
+  transition: transform 0.3s ease;
 }
-.mrb-message-read .mrb-message-element{
-    background: #ddd;
-    border: none;
-    box-shadow: none;
+.mrb-message-read .mrb-message-element {
+  background: #ddd;
+  border: none;
+  box-shadow: none;
 }
 .mrb-message-element.selected {
-    color: #ffffff;
-    background: #e25303;
+  color: #ffffff;
+  background: #e25303;
 }
-h5.mrb-message-title{
-	margin: 0;
+h5.mrb-message-title {
+  margin: 0;
 }
-.mrb-message-content-title{
-	margin: 0;
+.mrb-message-content-title {
+  margin: 0;
 }
-.mrb-message-date{
-	font-size: 16px;
-    font-weight: bold;
-	margin-bottom: 10px;
+.mrb-message-date {
+  font-size: 16px;
+  font-weight: bold;
+  margin-bottom: 10px;
 }
-.mrb-message-text-container{
-    height: calc(70vh);
-    padding: 20px;
+.mrb-message-text-container {
+  height: calc(70vh);
+  padding: 20px;
 }
-.mrb-message-text-container.no-img{
-    height: auto;
+.mrb-message-text-container.no-img {
+  height: auto;
 }
-.mrb-message-text{
-	font-size: 14px;
-	font-weight: bold;
+.mrb-message-text {
+  font-size: 14px;
+  font-weight: bold;
 }
 
 /*Gallery from design store*/
 .mrb-message-content .single-design-img-gallery .inner-container {
-    padding: 20px 20px 20px 20px;
-}
-.mrb-message-content .single-design-img-gallery .inner-container .design-img-container {
-    height: calc(70vh);
-}
-.mrb-message-content .single-design-img-gallery .inner-container .design-img-container.single-img-gallery {
-    margin-bottom: 0;
-    height: calc(70vh - 20px);
-}
-.mrb-message-content .single-design-img-gallery .inner-container .design-img-container #design-img-preview {
-    transition: background-image 0.1s ease-in-out 0.1s;
-    background-size: contain;
-    background-repeat: no-repeat;
-    background-position: center;
-    height: 100%;
-    text-align: center;
-}
-.mrb-message-content .single-design-img-gallery .inner-container .design-img-container #design-img-preview img {
-    visibility: hidden;
-    height: 100%;
+  padding: 20px 20px 20px 20px;
+}
+.mrb-message-content
+  .single-design-img-gallery
+  .inner-container
+  .design-img-container {
+  height: calc(70vh);
+}
+.mrb-message-content
+  .single-design-img-gallery
+  .inner-container
+  .design-img-container.single-img-gallery {
+  margin-bottom: 0;
+  height: calc(70vh - 20px);
+}
```

Comment:
for me this looks a bit weird, what do you think?

Reply:
As we discussed, Prettier is an opinionated formatter so in general there are few things you can change: https://prettier.io/docs/en/option-philosophy.html
(Same will happen with Black)